v7/import_blogger/import_blogger.py

```diff
@@ -182,21 +187,18 @@ def import_item(self, item, out_folder=None):
         else:
             is_draft = False
 
-        self.url_map[link] = self.context['SITE_URL'] + '/' + \
-            out_folder + '/' + slug + '.html'
-
+        self.url_map[link] = (self.context['SITE_URL'] + out_folder +
```

Comment:
It might be safer to ensure that SITE_URL ends with a '/' when populating the context.  Some blogs probably don't return a url ending in '/' and that may be why we had it in our code.